tests/legacy/generate_checkpoints.sh

```diff
@@ -1,44 +1,57 @@
 #!/bin/bash
-# Sample call:
-#  bash generate_checkpoints.sh 1.0.2 1.0.3 1.0.4
-
+# Usage:
+# 1. Generate checkpoints with one or more specified PL versions:
+#    bash generate_checkpoints.sh 1.0.2 1.0.3 1.0.4
+# 2. Generate checkpoints with the PL version installed in your environment:
+#    bash generate_checkpoints.sh
 set -e
 
-LEGACY_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
-FROZEN_MIN_PT_VERSION="1.6"
-
-echo $LEGACY_PATH
-# install some PT version here so it does not need to reinstalled for each env
-pip install virtualenv "torch==1.6" --quiet
-
-ENV_PATH="$LEGACY_PATH/vEnv"
+LEGACY_PATH=$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )
+ENV_PATH=$LEGACY_PATH/vEnv
+echo LEGACY_PATH: $LEGACY_PATH
+echo ENV_PATH: $ENV_PATH
 
 # iterate over all arguments assuming that each argument is version
-for ver in "$@"
+for pl_ver in "$@"
 do
-	echo "processing version: $ver"
-	# mkdir "$LEGACY_PATH/$ver"
-
-  # create local env
-  echo $ENV_PATH
-  virtualenv $ENV_PATH --system-site-packages
-  # activate and install PL version
-  source "$ENV_PATH/bin/activate"
+	echo processing version: $pl_ver
```

Comment:
nit: I suggest we avoid tabs, everywhere else we use spaces

Reply:
Thank you for pointing it out! It's been there since before and I missed it in my IDE 😄